protocol/x/affiliates/keeper/keeper.go

```diff
@@ -204,6 +212,10 @@ func (k Keeper) GetTierForAffiliate(
 		return 0, 0, err
 	}
 	tiers := affiliateTiers.GetTiers()
+	// Return 0 tier if no tiers are set.
+	if len(tiers) == 0 {
+		return 0, 0, errorsmod.Wrapf(types.ErrAffiliateTiersNotSet, "affiliate tiers are not set")
```

Comment:
Should we return `0, 0, nil` here to be safe? Thinking about the case where:
- Governance approves affiliate tiers
- Alice refers Bob
- Governane update affiliate tiers to empty
- No all taker trade by Bob will result in error (?)

By default returning a 0 tier and 0 fee share is not the most accurate but I believe fine practically, and also the safest. Really what we want to return is an additional variable `isAffiliateProgramActive`, but that's an overkill

Reply:
agreed